src/Panels/ChatLogPanel.cs

```diff
@@ -243,14 +297,14 @@ private void OnChatKeyDown(UIComponent component, UIKeyEventParameter eventParam
                 }
 
                 // If not connected to a server / hosting a server, tell the user and return
-                if (MultiplayerManager.Instance.CurrentRole == MultiplayerRole.None)
-                {
-                    PrintGameMessage(MessageType.Warning, "You can only use the chat feature when hosting or connected.");
-                    return;
-                }
+                //if (MultiplayerManager.Instance.CurrentRole == MultiplayerRole.None)
+                //{
+                //    PrintGameMessage(MessageType.Warning, "You can only use the chat feature when hosting or connected.");
+                //    return;
+                //}
```

Comment:
I commented out the need for users to not being able to use the chat box without being on a server. I did change the playerName to Local if they're not connected to a server.